crates/worker/src/partition/shuffle.rs

```diff
@@ -53,7 +55,7 @@ pub(crate) struct ShuffleInput(pub(crate) AckKind);
 pub(crate) enum PartitionProcessorMessage {
     Invocation(ServiceInvocation),
     Response(InvocationResponse),
-    Kill(FullInvocationId),
+    TerminateInvocation(InvocationTermination),
```

Comment:
Just `Terminate`?

Reply:
Will rename it into `InvocationTermination`.